.gitignore

```diff
@@ -4,7 +4,8 @@
 
 #Dependencies
 
-
+dump.rdb
```

Comment:
I think this is something redis related, as it only appeared when I started using Redis